.github/workflows/api.yml

```diff
@@ -4,9 +4,6 @@ on:
   push:
     branches:
     - master
-    paths:
-    - 'keyboards/**'
-    - 'layouts/community/**'
```

Comment:
This is now done for all changes because we expect to generate more data from core files right?

Reply:
It could instead modified to include:
* `data/`
* the 2 workflow files
* `lib/python` 
  * all as limiting this down would be a tad difficult to maintain
 
Was being lazy, though happy to swap it over.

Reply:
I'm still not quite up to speed on the api / data generation. But yes, if these are the single sources of truth we should only run the action for changes from these folders. :+1: